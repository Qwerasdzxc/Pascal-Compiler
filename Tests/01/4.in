191.761 717.918

