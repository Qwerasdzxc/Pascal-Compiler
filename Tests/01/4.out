909.68 -526.16 0.27